doc/storage_policy.md

```diff
@@ -0,0 +1,122 @@
+Storage Policy 
+==============
+
+Allows the enterprise admin to manage the Storage Policies for users in their
+enterprise. Used for an enterprise to decide storage location for users based on
+where they work/reside. 
+
+* [Get Storage Policy](#get-storage-policy) 
+* [Get List of Storage Policies](#get-list-of-storage-policies)
+* [Create New Assignment](#create-new-assignment)
+* [Get Assignment](#get-assignment)
+* [Get Assignment For Target](#get-assignment-for-target)
+* [Update Existing Assignment](#update-existing-assignment)
+* [Assign Storage Policy](#assign-storage-policy)
+* [Delete Assignment](#delete-assignment)
+
+Get Storage Policy
+------------------
+
+Calling [`getInfo(String fields ...)`][get-info] will return BoxStoragePolicy.Info object
+containing information about the storage policy. If necessary to retrieve 
+limited set of fields. It is possible to specify them using param.
+
+```java
+BoxStoragePolicy storagePolicy = new BoxStoragePolicy(api, id);
+BoxStoragePolicy.Info storagePolicyInfo = storagePolicy.getInfo();
+```
+
+[get-info]:http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxStoragePolicy.html#getInfo--
+
+Get List of Storage Policies
+----------------------------
+
+Calling the static [`getAll(BoxAPIConnection api)`][get-list-of-storage-policies]
+will return an iterable that will page through all of the storage policies. 
+It is possible to specify maximum number of items per response and fields to retrieve by 
+calling the static [`getAll(BoxAPIConnection api, int limit, String fields ...)`][get-list-storage-policies-with-fields] method. 
```

Comment:
This link is broken, looks like it doesn't match the identifier below with the URL